rivershared/cmd/update-mod-version/main.go

```diff
@@ -14,23 +18,9 @@ import (
 	"golang.org/x/mod/semver"
 )
 
-// Notably, `./cmd/river` is excluded from this list. Unlike the other modules,
-// it doesn't use `replace` directives so that it can stay installable with `go
-// install ...@latest`. Without `replace`, dependencies need a hard lock in
-// `go.sum`, so any changes to its `go.mod` file would require a `go mod tidy`
-// be run afterwards.
-var allProjectModules = []string{ //nolint:gochecknoglobals
-	".",
-	"./riverdriver",
-	"./riverdriver/riverdatabasesql",
-	"./riverdriver/riverpgxv5",
-	"./rivershared",
-	"./rivertype",
-}
```

Comment:
Nice: these bespoke lists of modules in various files like here and `Makefile` all go away.